pages/clusters/manage_queues.md

```diff
@@ -31,15 +40,20 @@ To create a new queue using the Buildkite interface:
 1. Select **Agents** in the global navigation to access the **Clusters** page.
 1. Select the cluster in which to create the new queue.
 1. On the **Queues** page, select **New Queue**.
-1. Enter a **key** for the queue, which can only contain letters, numbers, hyphens, and underscores, as valid characters.
+1. In the **Create a key** field, enter a unique _key_ for the queue, which can only contain letters, numbers, hyphens, and underscores, as valid characters.
 1. Select the **Add description** checkbox to enter an optional longer description for the queue. This description appears under the queue's key, which is listed on the **Queues** page, as well as when viewing the queue's details.
+1. In the **Select your compute** section, select between [**Hosted**](/docs/pipelines/hosted-agents/overview) or **Self hosted** for your agent infrastructure.
+1. If you chose **Hosted**, complete the remaining sub-steps:
+    1. In the new **Configure your hosted compute** section, select your **Machine type** ([**Linux**](/docs/pipelines/hosted-agents/linux) or [**macOS**](/docs/pipelines/hosted-agents/mac)).
+    1. If you selected **Linux**, within **Architecture**, you can choose between **AMD64** (the default and recommended) or **ARM64** architectures for the Linux machines running as hosted agents. To switch to **ARM64**, select **Change**, followed by **ARM64 (AArch64)**.
```

Comment:
```suggestion
    1. If you selected **Linux**, within **Architecture**, you can choose between **AMD64** (the default) or **ARM64** architectures for the Linux machines running as hosted agents. To switch to **ARM64**, select **Change**, followed by **ARM64 (AArch64)**.
```

We don't have a recommended architecture.

Reply:
Then this part of the UI should change:

![Screenshot 2024-05-13 at 2 46 18 PM](https://github.com/buildkite/docs/assets/23667807/9f082e71-bee4-4f22-ab8b-6214f6e95d06)